rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -196,6 +196,18 @@ void ObjectRenderer::renderInstance(InstanceObject *instance,
 		return;
 	}
 
+
+	// Handles times provided by TOBJ data
+	const auto currentHour = m_world->getHour();
+	if (instance->object->timeOff < instance->object->timeOn)
+	{
+		if ( currentHour >= instance->object->timeOff && currentHour < instance->object->timeOn )
+			return;
+	} else {
+		if ( currentHour >= instance->object->timeOff || currentHour < instance->object->timeOn )
+			return;
+	}
```

Comment:
nit:  I guess I'd have save some bool result if the time is in the timeOn / timeOff range (`isInRange`) and then checked if `drawIfInRange` is also true. I prefer if variables are used because their name can explain what the intention is.